source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -782,6 +787,8 @@ thread SHOULD
   Timeout: Background Connection Pooling
   <../client-side-operations-timeout/client-side-operations-timeout.rst#background-connection-pooling>`__.
 
+A pool SHOULD allow force scheduling the next background thread iteration to remove perished connections including "in use" connections. 
```

Comment:
```suggestion
A pool SHOULD allow immediate scheduling of the next background thread iteration.
```
The "SHOULD" here applies to the scheduling behavior, the work done by the thread during the next run is unchanged.